be/src/exec/hdfs_scanner.h

```diff
@@ -343,11 +342,12 @@ struct HdfsScannerContext {
 
     bool orc_use_column_names = false;
 
-    bool can_use_any_column = false;
-
     bool can_use_min_max_count_opt = false;
 
+    bool is_first_split = false;
     bool return_count_column = false;
+    bool emit_count_column = false;
```

Comment:
What is the purpose of the emit_count_column?

Where is this parameter set in FE?

Reply:
it's just local variable.

because when calling get_next
1. output chunk
2. output eof